config/routes.rb

```diff
@@ -16,6 +16,9 @@
       get 'find_policy_list'
       get 'category' =>  'policies#category_fields'
```

Comment:
@GauravMakwana-BTC : All the policy routes should come in scope or create collection or member routes for this